bin/yarn

```diff
@@ -4,8 +4,8 @@ Dir.chdir(APP_ROOT) do
   begin
     exec "yarnpkg", *ARGV
   rescue Errno::ENOENT
-    $stderr.puts "Yarn executable was not detected in the system."
-    $stderr.puts "Download Yarn at https://yarnpkg.com/en/docs/install"
+    warn "Yarn executable was not detected in the system."
+    warn "Download Yarn at https://yarnpkg.com/en/docs/install"
```

Comment:
Weird, I think this is Rails default

Reply:
I can revert it and tell rubocop to ignore it if we object, but I figured it wasn't a big deal given we don't... use... yarn... anyway...

Reply:
Probably OK, we just might end up reverting it with the next `rails app:update`